connectors/redis/redis.go

```diff
@@ -40,6 +41,19 @@ type SimpleRedis interface {
 	Shutdown() error
 }
 
+// Scope is a namespace wrapper around a stats reporter, ensuring that
+// all emitted values have a given prefix or set of tags.
+type Scope interface {
```

Comment:
`Scope` isn't something that's tied to the redis connector specifically. We may have other connectors where we want to do logging as well. I would say let's put this interface in it's own package.